Sources/App/Models/Reference.swift

```diff
@@ -56,6 +56,14 @@ enum Reference: Equatable, Hashable {
                 return tagName
         }
     }
+
+    var versionKind: Version.Kind {
+        if let semVer = semVer {
+            return semVer.isPreRelease ? .preRelease : .release
+        } else {
+            return .defaultBranch
+        }
```

Comment:
```suggestion
        switch self {
            case .branch:
                return .defaultBranch
            case .tag(let semVer, _):
                return semVer.isPreRelease ? .preRelease : .release
        }
```

I think that's a little clearer to the reader that this unwraps the cases, rather than going through the `semVer` accessor indirectly.